src/NuProj.Tasks/Extensions.cs

```diff
@@ -143,5 +157,24 @@ public static void UpdateMember<T>(this T target, Expression<Func<T, string>> me
             
             property.SetValue(target, list, null);
         }
+
+        public static string Combine(this PackageDirectory packageDirectory, string targetFramework, string fileName)
+        { 
+            switch (packageDirectory)
+            {
+                case PackageDirectory.Root:
+                    return fileName;
+                case PackageDirectory.Content:
+                    return Path.Combine(Constants.ContentDirectory, fileName);
+                case PackageDirectory.Build:
+                    return Path.Combine(Constants.BuildDirectory, targetFramework, fileName);
```

Comment:
It's intentional, sort of. Docs (https://docs.nuget.org/create/creating-and-publishing-a-package#import-msbuild-targets-and-props-files-into-project) allow to do per-framework targets. To be have a consistent behavior, I opted to use "build\framwork" folder.

In retrospect, until we have a way to change/suppress automatic target framework resolution, it might be better to use "build" folder only. I'll try to patch this later this evening.